addons/mail/static/src/new/web/messaging_menu/notification_item.js

```diff
@@ -2,7 +2,7 @@
 
 import { Component, useRef } from "@odoo/owl";
 import { PartnerImStatus } from "@mail/new/discuss/partner_im_status";
-import { RelativeTime } from "../core_ui/relative_time";
+import { RelativeTime } from "../../core_ui/relative_time";
```

Comment:
It's too complex, I would use absolute path (well I would always use absolute path personally).

Reply:
I will change them all into absolute paths then( still fixing red runbot)